apps.awesim.org/apps/bc_desktop/ascend.yml.erb

```diff
@@ -0,0 +1,74 @@
+<%-
+  groups = OodSupport::User.new.groups.sort_by(&:id).tap { |groups|
+    groups.unshift(groups.delete(OodSupport::Process.group))
+  }.map(&:name).grep(/^P./)
+-%>
+---
+title: "Ascend Desktop"
+cluster: "ascend"
+description: |
+  This app will launch an interactive desktop on one or more compute nodes. It is
+  a large environment for when you need a lot of compute and/or memory resources because 
+  you will have full access to all the resources on that compute node(s).
+  
+  If you do not need all these resources, use the 
+  [Lightweight Desktop](/pun/sys/dashboard/batch_connect/sys/bc_desktop/vdi/session_contexts/new)
+  app instead which is much more lightweight for general-purpose use cases.
+form:
+  # everything is taken from bc_desktop/form.yml except cores is added
+  - bc_vnc_idle
+  - desktop
+  - account
+  - bc_num_hours
+  - cores
+  - bc_num_slots
+  - licenses
+  - node_type
+  - bc_queue
+  - bc_vnc_resolution
+  - bc_email_on_started
+attributes:
+  desktop:
+    widget: select
+    label: "Desktop environment"
+    options:
+      - ["Xfce", "xfce"]
+      - ["Mate", "mate"]
+    help: |
+      This will launch either the [Xfce] or [Mate] desktop environment on the
+      [Ascend cluster].
+
+      [Xfce]: https://xfce.org/
+      [Mate]: https://mate-desktop.org/
+      [Ascend cluster]: https://www.osc.edu/resources/technical_support/supercomputers/ascend
+  bc_queue: null
+  account:
+    label: "Project"
+    widget: select
+    options:
+      <%- groups.each do |group| %>
+      - "<%= group %>"
+      <%- end %>
+  cores:
+    widget: number_field
+    value: 46
+    min: 1
+    max: 88
+    step: 1
+  licenses:
+    help: |
+      Licenses are comma separeated in the format '\<name\>@osc:\<# of licenses\>' like
+      'stata@osc:1'. More help can be found on our website regarding the changes
+      in the [slurm migration].
+
+      [slurm migration]: https://www.osc.edu/resources/technical_support/supercomputers/owens/environment_changes_in_slurm_migration
+    pattern: '^([\w]+@osc:\d+,{0,1})+$'
+  node_type:
+    widget: select
+    label: "Node type"
+    help: |
+      Need docs
+    options:
+      - [ "any", "any" ]
```

Comment:
I wonder if it's better to offer CPU vs GPU.  The "Any" (ie not asking for GPU) will get CPU only job where max cores is 84.

Reply:
Done!